### PR TITLE
feed grid styling

### PR DESCRIPTION
**Pull request by claireclark1**
Purpose
---------------
feature
[https://www.pivotaltracker.com/story/show/179060216](url)

Solution
---------------
style feed grid according to [https://app.zeplin.io/project/60a3f1172759472a903908f7/screen/61003f320349c51086d0fc63](url)

Change summary
---------------
* add masonry layout
* adjust the other styling as needed
* mock data that is currently null

Steps to Verify
----------------
1. run code
2. check styling

Additional details / screenshot
----------------
Screen Shot 2021-08-09 at 11 28 39 AM
Screen Shot 2021-08-09 at 11 28 50 AM

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -15,6 +15,10 @@
 @import "./components/scss/ActionsBar";
 @import "./components/scss/PostList";
 
+.App {
+  overflow-x: hidden;
+}
```
Comment (claireclark1) [inline src/App.scss]: move into touch-ups story.  not relevant to feed.

File: src/components/Post.tsx [hunk 2]
```diff
@@ -14,40 +14,34 @@ interface PostProps {
 }
 
 const Post = ({ feedItem }: PostProps): JSX.Element => {
-  const [showActionsBar, setShowActionsBar] = useState<boolean>(false);
   const noteContent = feedItem.content;
+  const userAddress = noteContent.content.split(" ").pop();
```
Comment (claireclark1) [inline src/components/Post.tsx]: delete

File: src/components/Profile.tsx [hunk 3]
```diff
@@ -14,6 +14,8 @@ const Profile = (): JSX.Element => {
 
   const handle = "Hans";
   const profileName = "lovetoeat";
+  const avatar =
```
Comment (claireclark1) [inline src/components/Profile.tsx]: delete

File: src/components/UserAvatar.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Avatar } from "antd";
 import { HexString } from "../utilities/types";
+import * as blockies from "blockies-ts";
```
Comment (claireclark1) [inline src/components/UserAvatar.tsx]: revert to no changes

File: src/services/sdk.ts [hunk 5]
```diff
@@ -193,7 +193,9 @@ const dispatchFeedItem = (
 ) => {
   const decoder = new TextDecoder();
 
-  const timestamp = content.published ? Date.parse(content.published) : 0;
```
Comment (claireclark1) [inline src/services/sdk.ts:196]: delete

File: src/theme/antd-overrides.scss [hunk 6]
```diff
@@ -62,7 +62,10 @@ a {
 }
 
 .ant-card.Post__block {
-  margin: 8px 0;
```
Comment (claireclark1) [inline src/theme/antd-overrides.scss:65]: revert to OG

File: src/utilities/activityContent.ts [hunk 7]
```diff
@@ -31,10 +31,6 @@ const createMediaAttachment = (item: string): ActivityContentAttachment => {
         return createVideoAttachment([
           createVideoLink(item, "video/mp4", activityContentHashes),
         ]);
-      case "wmv":
```
Comment (claireclark1) [inline src/utilities/activityContent.ts]: revert to OG

File: src/components/PostList.tsx [hunk 8]
```diff
@@ -47,34 +48,37 @@ const PostList = ({ feedType }: PostListProps): JSX.Element => {
...
-    <div className="PostList__block">
-      {currentFeed.length > 0 ? (
-        <>
-          {currentFeed
-            .slice(0)
-            .reverse()
-            .map((post, index) => {
-              if (!post.fromAddress)
-                throw new Error(`no fromAddress in post: ${post}`);
+  currentFeed = currentFeed.sort(function (a, b) {
```
Comment (enddynayn) [inline src/components/PostList.tsx]: I think you dont need a variable here sort mutates the original array

File: src/components/PostMedia.tsx [hunk 9]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useEffect } from "react";
```
Comment (enddynayn) [inline src/components/PostMedia.tsx]: did you add useEffect on accident? look like you are not using it.

**Review by enddynayn — APPROVED** [review]

**Pull request merged**